From: =?utf-8?q?Petr_Van=C4=9Bk?= <pv@excello.cz>
Date: Fri, 13 Dec 2019 10:32:22 +0100
Subject: Append CXX to the list of needed languages

The replxx is written in C++ and it is enabled by default. We need to
specify this to correctly pass user specific CXXFLAGS.
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 39a1773..067ab26 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -21,7 +21,7 @@ ENDIF()
 
 SET(RSPAMD_VERSION         "${RSPAMD_VERSION_MAJOR}.${RSPAMD_VERSION_MINOR}")
 
-PROJECT(rspamd VERSION "${RSPAMD_VERSION}" LANGUAGES C ASM)
+PROJECT(rspamd VERSION "${RSPAMD_VERSION}" LANGUAGES C CXX ASM)
 
 # This is supported merely with cmake 3.1
 SET(CMAKE_C_STANDARD 11)
